nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -467,6 +467,7 @@
                         <exclude>src/test/resources/TestUpdateRecord/output/person-with-firstname.json</exclude>
                         <exclude>src/test/resources/TestUpdateRecord/output/person-with-firstname-lastname.json</exclude>
                         <exclude>src/test/resources/TestUpdateRecord/output/person-with-capital-lastname.json</exclude>
+                        <exclude>src/test/resources/TestUpdateRecord/output/person-with-bool.json</exclude>
```

Comment:
This should be in the TestConvertJSONToSQL section rather than TestUpdateRecord, it causes a RAT failure